### ISSUE TITLE
Message "Docker file is damaged" when downloading Docker Mac RC3

### ISSUE THREAD

**Issue by ritamargherita** [docker/for-mac#5566]
- [ ] I have tried with the latest version of Docker Desktop
  - [ ] I have tried disabling enabled experimental features
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID:

### Information

  - macOS Version:
  - Docker Desktop Version:

### Steps to reproduce the behavior

I had the Mac RC1 that was working fine. Today Docker was frozen and not working at all, so I have deleted the app and tried to download the latest RC. Problem is that I can't install it because every time I try I get an error message saying that the file is damaged and can't be opened.

**Comment by stephen-turner** [comment]
Hmmm, I can't explain that. I don't think it can be damaged at our end because we have thousands of users running it and no-one else has reported this problem.

**Comment by ritamargherita** [comment]
Can you please provide me the link to download the RC3 please? I just tried again now from here: https://desktop.docker.com/mac/stable/arm64/62632/Docker.dmg and it keeps saying that the file is damaged

**Comment by stephen-turner** [comment]
That's the correct link. Do you have any sort of proxy that could be modifying it?

**Comment by ritamargherita** [comment]
I have restarted my Mac and now it works fine. Not sure where the problem was, but it is fixed now! Thanks anyway for your time

**Comment by stephen-turner** [comment]
OK, good, thanks for letting us know.

**Comment by ajardin** [comment]
Same issue on my side with an MBP 2017 and macOS 11.2.3.

I faced several crashes with the Docker Desktop application today. The application ended up being completely frozen. So I removed it to download a newer version. And like @ritamargherita, I started to encounter the "damaged app" issue during the install process. I tried both https://desktop.docker.com/mac/stable/Docker.dmg and https://desktop.docker.com/mac/edge/Docker.dmg without any success, even after a computer restart.

@stephen-turner is it possible to download another version?

**Comment by ajardin** [comment]
I guess the crash I saw yesterday had some impacts on the OS itself... I tried several versions (`3.3.0`, `3.2.2`, `3.2.1`, `3.2.0`) from the [releases page](https://docs.docker.com/docker-for-mac/release-notes/) with the same results. I ended up dropping this approach and trying Homebrew. It worked immediately.

**TL;DR:** if the GUI install process is broken for no reason, you could try Homebrew.

```
$ brew install --cask docker
```
